cockroachdb/example/cert-manager/certificates-patch.yaml

```diff
@@ -87,3 +88,27 @@ spec:
                     - key: tls.key
                       path: client.root.key
             defaultMode: 256
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cockroach-backup-init
+spec:
+  template:
+    spec:
+      volumes:
+        - name: client-certs
+          secret:
+            $patch: delete
+          projected:
+            sources:
+              - secret:
+                  name: cockroachdb.client.root
+                  items:
+                    - key: ca.crt
```

Comment:
Wondering if we can move these patches(including the ones for the stateful set) inside the base.